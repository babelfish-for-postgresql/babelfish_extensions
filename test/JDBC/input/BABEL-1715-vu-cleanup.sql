USE MASTER
GO

drop table bbl_1715_t1;
go
drop table bbl_1715_t2;
go
drop table cx_bbl_1715;
go
