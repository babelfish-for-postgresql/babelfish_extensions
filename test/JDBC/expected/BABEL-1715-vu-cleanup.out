USE MASTER
GO

drop table babel_1715_vu_prepare_t1;
go
drop table babel_1715_vu_prepare_t2;
go
drop table cx_babel_1715_vu_prepare;
go
